content/source/docs/cloud/integrations/kubernetes/index.html.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: "cloud"
+page_title: "Terraform Cloud Operator for Kubernetes setup instructions"
+description: |-
+  HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official Terraform Cloud Operator for Kubernetes to provision infrastructure through Terraform Cloud using their Kubernetes control plane
+---
+
+# Terraform Cloud Operator for Kubernetes Setup Instructions
+
+~> **Important:** The Terraform Cloud Operator for Kubernetes is still under development and in the alpha testing stage. It is not ready for production use.
+
+## Overview
+
+HashiCorp Terraform Cloud customers can integrate with Kubernetes using the official [Terraform Cloud Operator for Kubernetes](https://github.com/hashicorp/terraform-k8s) to provision infrastructure internal or external to the Kubernetes cluster directly from the Kubernetes control plane.  Using the Terraform Cloud Operator for Kubernetes' CustomResourceDefinition (CRD), users can dynamically create Terraform Cloud workspaces (using a Terraform configuration from a git repository or from the [Terraform Registry](https://registry.terraform.io/), populate variables, and perform Terraform runs to provision infrastructure.
+
+## Prerequisites
+
+All Terraform Cloud users can use the Terraform Cloud Operator for Kubernetes. Some features of Terraform Cloud that are [limited to certain tiers](/docs/cloud/paid.html) aren't available to the Terraform Cloud Operator for Kubernetes unless you've purchased the corresponding tier.
+
+## Terraform Cloud Operator for Kubernetes
+
+### Networking Requirements
+
+In order for the Terraform Cloud Operator for Kubernetes to function properly, it must be able to make outbound requests over HTTPS (TCP port 443) to the Terraform Cloud application APIs. This may require perimeter networking as well as container host networking changes, depending on your environment. The IP ranges are documented in the [Terraform Cloud IP Ranges documentation](/docs/cloud/architectural-details/ip-ranges.html). The services which run on these IP ranges are described in the table below.
+
+Hostname | Port/Protocol | Directionality | Purpose
+ --|--|--|--
+ app.terraform.io | tcp/443, HTTPS | Outbound | Dynamically managing Terraform Cloud Workspaces and returning the output to Kubernetes via the Terraform Cloud API
+
+### Compatibility
+
+The current release of the Terraform Cloud Operator for Kubernetes supports the following versions:
+
+* Kubernetes 1.14 and above
+
+### Installation & Configuration
+
+1. Generate an [organization token](/docs/cloud/users-teams-organizations/api-tokens.html#organization-api-tokens) within Terraform Cloud and save it to a file. (These instructions assume you're using a file named `credentials`.)
+
+1. Create a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/) with the Terraform Cloud API credentials
```

Comment:
Still need to add a period at the sentence end, for this list item and the two after it.

Reply:
Should we add text here on what a credentials file looks like? Or perhaps link to https://www.terraform.io/docs/commands/cli-config.html#credentials-1